### PR TITLE
Mwpw-131609 mp4 seo test

### PR DESCRIPTION
**Pull request by vhargrave**
Fix https://jira.corp.adobe.com/browse/MWPW-131609
Allows for structured video data to be added to a page, to improve on SEO.

Test URLs:
- Before: https://main--express-website--adobe.hlx.page/drafts/vhargrave/mov-to-mp4?lighthouse=on
- After: https://mwpw-131609-mp4-seo-test--express-website--adobe.hlx.page/drafts/vhargrave/mov-to-mp4?lighthouse=on

To test the structured data you can go here - https://search.google.com/test/rich-results/result/r%2Fvideos?id=A1F58mx5rtCaUggVr5xu6w

This implementation is borrowed from milo and has been only slightly adapted for express.

### CODE REVIEW

**Comment by vhargrave** [comment]
This PR has been created off the change to the [unit test framework PR](https://github.com/adobe/express-website/pull/864), so that unit tests could be used from milo. Wait for the other PR to be merged first, before merging this one.

**Review by qiyundai — APPROVED** [review]
Presuming the console.log's are all intentional, lgtm!
The linting errors should have been fixed in a different PR.

**Pull request merged**
